sys/vfs_syscalls.c

```diff
@@ -85,6 +88,17 @@ int do_fstat(thread_t *td, int fd, vattr_t *buf) {
   return res;
 }
 
+int do_mount(thread_t *td, const char *fs, const char *path) {
```

Comment:
We would want to extend interface of mount to have extra flags argument, so we can have 'read only', 'no exec' and possibly other types of mounts. Not a big deal and can be added in future.